salsa_dancing_molecules/simulations/simulation.py

```diff
@@ -8,10 +72,46 @@ def run(sim_info, atoms):
         sim_info - dictionary with information on the simulation.
         atoms - atoms object to be used.
     """
-    # TODO: implement real simulation
-    f = open(sim_info["traj_output_path"], "w")
-    f.write(sim_info["potential"])
-    f.close()
-    f = open(sim_info["csv_output_path"], "w")
-    f.write(sim_info["ensemble"])
-    f.close()
+    sim_info["use-asap"] = (sim_info["use-asap"].lower() == "true")
+    choose_potential(sim_info["potential"],
+                     sim_info["kim-model"],
+                     sim_info["use-asap"],
+                     atoms)
+
+    # Set the momenta corresponding to set temperature
```

Comment:
Maybe "Initialise the momenta from the chosen initial temperature" would be a better phrasing?